docs/tutorials/orc.ipynb

```diff
@@ -0,0 +1,336 @@
+        "label_cols = [\"species\"]\n",
+        "\n",
+        "# select feature columns\n",
+        "feature_dataset = tfio.IODataset.from_orc(\"iris.orc\", columns=feature_cols)\n",
+        "# select label columns\n",
+        "label_dataset = tfio.IODataset.from_orc(\"iris.orc\", columns=label_cols)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "GSYMP48vVvV0"
+      },
+      "source": [
+        "A util function to map species to float numbers for model training:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "TQvuE7OgVs1q"
+      },
+      "outputs": [],
+      "source": [
+        "@tf.function\n",
+        "def species_float_conversion(x):\n",
+        "    if x == \"virginica\":\n",
+        "        return 1.0\n",
+        "    if x == \"versicolor\":\n",
+        "        return 2.0\n",
+        "    if x == \"setosa\":\n",
+        "        return 3.0\n",
+        "    return 4.0\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "lpf0w41iWAZ4"
+      },
+      "outputs": [],
+      "source": [
+        "label_dataset = label_dataset.map(species_float_conversion)\n",
+        "dataset = tf.data.Dataset.zip((feature_dataset, label_dataset))\n",
+        "dataset = dataset.batch(1)\n",
+        "\n",
+        "def pack_features_vector(features, labels):\n",
+        "    \"\"\"Pack the features into a single array.\"\"\"\n",
+        "    features = tf.stack(list(features), axis=1)\n",
+        "    return features, labels\n",
+        "\n",
+        "dataset = dataset.map(pack_features_vector)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "R1Tyf3AodC2Y"
+      },
+      "source": [
+        "## Build, compile and train the model"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oVB9Q0B-WDn4"
+      },
+      "source": [
+        "Finally, you are ready to build the model and train it! You will build a 3 layer keras model to predict the class of the iris plant from the dataset you just processed."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "tToy0FoOWG-9"
+      },
+      "outputs": [],
+      "source": [
+        "model = tf.keras.Sequential(\n",
+        "    [\n",
+        "        tf.keras.layers.Dense(\n",
+        "            10, activation=tf.nn.relu, input_shape=(4,)\n",
+        "        ),  # input shape required\n",
+        "        tf.keras.layers.Dense(10, activation=tf.nn.relu),\n",
+        "        tf.keras.layers.Dense(3),\n",
+        "    ]\n",
+        ")\n",
+        "\n",
+        "model.compile(optimizer=\"adam\", loss=\"binary_crossentropy\", metrics=[\"accuracy\"])\n",
```

Comment:
> loss=\"binary_crossentropy\"

BInary crossentrpy is not the right loss here. You probably want `CategoricalCrossentropy(from_logits=True)`.